pkg/operator/ceph/file/mds/livenessprobe.go

```diff
@@ -75,8 +76,6 @@ func generateMDSLivenessProbeExecDaemon(daemonID, filesystemName, keyring string
 		ProbeHandler: v1.ProbeHandler{
 			Exec: &v1.ExecAction{
 				Command: []string{
-					"timeout",
-					fmt.Sprintf("%d", mdsCmdTimeout),
```

Comment:
`timeout` can keep the script from running too long, but it exits with error code after timeout, which we don't want.